### PR TITLE
[Model Monitoring] Improve the error messages in drift table creation

### PR DESCRIPTION
**Pull request by jond01**
Fixes [ML-5265](https://jira.iguazeng.com/browse/ML-5265).

This change covers two cases:
- Extra features in the inputs (current data), as in the original issue.
- Missing keys, e.g. "mean", from existing features in the inputs. This happens when there's a placeholder for a label, but a new label name is used.

### CODE REVIEW

File: mlrun/model_monitoring/features_drift_table.py [hunk 1]
```diff
@@ -93,7 +93,6 @@ def __init__(self):
 
     def produce(
         self,
-        features: list[str],
```
Comment (Eyal-Danieli) [inline mlrun/model_monitoring/features_drift_table.py:96]: remove `features` from the docstring as well

File: mlrun/model_monitoring/features_drift_table.py [hunk 2]
```diff
@@ -279,6 +278,7 @@ def _plot_feature_row_table(
         input_statistics: dict,
         metrics: dict,
         row_height: int,
+        feature: str,
```
Comment (Eyal-Danieli) [inline mlrun/model_monitoring/features_drift_table.py:281]: add it to the docstring to keep consistency with the other function params

Comment (Eyal-Danieli) [reply mlrun/model_monitoring/features_drift_table.py:281]: actually why do you prefer to pass `feature` here if you already have `feature_name`? perhaps it's better to pass only `feature_name` and change the variable name when parsing it into html format. something like 
`parsed_feature_name = "<br>".join(self._separate_feature_name(feature_name))`

**Review by Eyal-Danieli — COMMENTED** [review]
nice
few little comments/suggestions

File: mlrun/model_monitoring/features_drift_table.py (hunk 1 above)
Comment (jond01) [reply mlrun/model_monitoring/features_drift_table.py:96]: a79ba82

File: mlrun/model_monitoring/features_drift_table.py (hunk 2 above)
Comment (jond01) [reply mlrun/model_monitoring/features_drift_table.py]: Right, great catch! 😅
Done in c6f8e0d.

**Review by Eyal-Danieli — APPROVED** [review]

**Pull request merged**